### PR TITLE
More Opinionated Updates

### PR DESCRIPTION
**Pull request by codecaffeine**
Spacing updates with curly braces.
Empty Option Sets

### CODE REVIEW

**Comment by ryang1428** [comment]
:+1:

**Comment by ndis1** [comment]
+1

**Comment by rmigneco** [comment]
:rocket:

File: README.md [hunk 1]
```diff
@@ -51,6 +58,8 @@ enum Planet {
 }
 ```
 
+- If there are no options, use the empty option set `[]` rather than the enum raw value `NSLayoutFormatOptions(rawValue: 0)`.
```
Comment (jgrandelli) [inline README.md]: I feel like there is a better place for this than `#Naming`. Or maybe not.

Comment (codecaffeine) [inline README.md]: `¯\_(ツ)_/¯` I do want to avoid a `#Misc` section

Comment (ryang1428) [inline README.md]: maybe a `#Autolayout` section if we will eventually have some more in there ?

Comment (codecaffeine) [inline README.md]: Well, empty option sets apply to other things than autolayout

Comment (ryang1428) [inline README.md]: trudat

Comment (ndis1) [inline README.md]: how about an `enum` section

Comment (jgrandelli) [inline README.md]: I second this an enum & struct section would be greatly appreciated

Comment (codecaffeine) [inline README.md]: I just put it under `# Types` for now since Sets are defined by a protocol and can be conformed to by classes, structs, and enum

**Pull request merged**